app/assets/stylesheets/components/_forewords.scss

```diff
@@ -0,0 +1,47 @@
+$gutter-small: 16px;
+$gutter: 40px;
+
+.forewords {
```

Comment:
<img width="1051" alt="image" src="https://user-images.githubusercontent.com/56026099/120784284-d4d22080-c523-11eb-8df9-e1305f2cbe50.png">
<img width="1441" alt="image" src="https://user-images.githubusercontent.com/56026099/120784383-ee736800-c523-11eb-9f3b-95e3e3d2c472.png">

I've been working on trying to implement these changes but it's caused some issues moving the padding over to the section. With padding on the section itself, the background of the foreword boxes no longer extends to the edge of the page.

Could add negative margin to those boxes so they extend to the edge of the page? I haven't committed this change for the moment

Reply:
Probably best to avoid negative margin. 

Is .section--forewords on the same element as .forewords or is it on a wrapper element? You'll want to background-color to be on the element that has the padding I think. I think this means moving the background colour to the section element.

Happy to look in more detail if needed :)